src/commands/linkAdmin.ts

```diff
@@ -0,0 +1,128 @@
+import { logger } from "../logger";
+import {
+  inlineCode,
+  SlashCommandBuilder,
+  SlashCommandStringOption,
+  SlashCommandSubcommandBuilder,
+} from "@discordjs/builders";
+import { CommandInteraction, CacheType, AutocompleteInteraction, MessageActionRow, MessageSelectMenu } from "discord.js";
+import { CommandType } from "../types";
+import fs from "fs";
+import { Link, linkModel } from "../schemas/link";
+import path from "path";
+import { CreateNewLink, FindLinkByName, GetAllShortenLinks } from "../helpers/linkQueries";
+
+
+const linkAdminModule: CommandType = {
+  data: new SlashCommandBuilder()
+    .setName("link-admin")
+    .setDescription("Which link?")
+    .addSubcommand(subcommand =>
+      subcommand
+        .setName("create")
+        .setDescription("Add new link?")
+        .addStringOption(option =>
+          option
+            .setName("shorten_link")
+            .setDescription("What is the name of the link?")
+            .setRequired(true)
+        )
+        .addStringOption(option =>
+          option
+            .setName("url")
+            .setDescription("What is the url of the link?")
+            .setRequired(true)
+        )
+    )
+    .addSubcommand(subcommand =>
+      subcommand
+        .setName("delete")
+        .setDescription("delete link?")
+        .addStringOption((option: SlashCommandStringOption) => {
+          option
+            .setName("link")
+            .setDescription("Choose Link")
```

Comment:
You can be a bit more descriptive with option descriptions -  it's a good idea to state what will happen when to your choice. In this case, "Select a link to delete" would be very good.